index.d.ts

```diff
@@ -125,6 +125,30 @@ declare namespace XLSX {
     DOUBLE_ACCOUNTING_UNDERLINE = 4,
   }
 
+  interface DataValidation {
```

Comment:
Move this after `interface ChartFont`. The XLSX namespace is used to isolate enums/classes that we only need the types of.